tests/http-adapter.spec.js

```diff
@@ -1,198 +1,243 @@
 import HttpAdapter from '../src/http-adapter'
-import sinon from 'sinon'
 
-describe('HTTP adapter', function () {
-  let adapter, server
+describe('HTTP Adapter', function () {
```

Comment:
It's been a while since I revisited these tests, which leads me to believe there is a better way to test this. Will look into but going to open this as a draft to get feedback